tests/osn-tests/src/test_nodeobs_api.ts

```diff
@@ -46,7 +47,7 @@ describe('nodebs_api', () => {
             try {
                 stats = osn.NodeObs.OBS_API_getPerformanceStatistics();
             } catch(e) {
-                throw new Error("OBS_API_getPerformanceStatistics threw an expection");
+                throw new Error(getCppErrorMsg(e));
```

Comment:
I would put these kind of changes in a separate PR to facilitate the work for the reviewer.

Reply:
Sounds good for this one 👍